seeker/pet-details-pending.html

```diff
@@ -0,0 +1,176 @@
+              <li><a href="login.html" class="nav__link">Shelters</a></li>
+              <li><a href="#" class="nav__link">Donate</a></li>
+            </ul>
+          </div>
+          <div class="nav__icons">
+            <i class="material-icons">search</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons">account_circle</i>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <div class="main__wrapper">
+            <div class="details-container">
+                <div class="details-item pet-details-card">
+                    <div class="pet-profile-container">
+                        <div class="pet-profile-photos">
+                            <div class="photo main-photo">
+                                <img src="../img/logo_ref.png" alt="">
+                            </div>
+                            <div class="photo-row">
+                                <div class="photo small-photo selected"><img src="../img/logo_ref.png" alt=""></div>
+                                <div class="photo small-photo"><img src="../img/logo_ref.png" alt=""></div>
+                                <div class="photo small-photo"><img src="../img/logo_ref.png" alt=""></div>
+                                
+                            </div>
+                        </div>
+                        <div class="pet-profile-details">
+                            <div class="pet-profile-header">
+                                <div>
+                                    <div class="title">Pet Name</div>
+                                    <div class="address">1234 Nowhere St. Toronto, ON</div>
+                                </div>
+                                <svg class="heart-icon" xmlns="http://www.w3.org/2000/svg" width="35" height="35" viewBox="0 0 61 54" fill="none">
+                                    <path fill-rule="evenodd" clip-rule="evenodd" d="M30.5358 7.8639L27.3674 4.6955C21.1068 -1.56517 10.9562 -1.56517 4.69552 4.6955C-1.56517 10.9562 -1.56517 21.1068 4.69552 27.3675L30.4591 53.131L30.4644 53.1257L30.541 53.2024L56.3045 27.4389C62.5652 21.1782 62.5652 11.0277 56.3045 4.76703C50.044 -1.49367 39.8935 -1.49367 33.6328 4.76703L30.5358 7.8639ZM30.4644 44.8815L44.8525 30.4931L49.0813 26.4094L49.0856 26.4135L52.1823 23.3168C56.1666 19.3327 56.1666 12.8733 52.1823 8.88919C48.1984 4.90513 41.7388 4.90513 37.7549 8.88919L30.539 16.1051L30.5177 16.084L30.5148 16.0871L23.2453 8.81769C19.2612 4.8336 12.8017 4.8336 8.81768 8.81769C4.83359 12.8017 4.83359 19.2612 8.81768 23.2454L16.2196 30.6473L16.2249 30.642L30.4644 44.8815Z" fill="black"/>
+                                </svg>
+                            </div>
+                            <div class="pet-profile-info">
+                                <div class="row">
+                                    <div class="key">Shelter:</div>
+                                    <div class="value">Some Shelter</div>
```

Comment:
Very minor detail but I think Yanbo used "South Shelter" as the shelter names, maybe for consistency we can change it so we can link the pages together.